api/comments.rb

```diff
@@ -1,3 +1,32 @@
+get "#{APIPREFIX}/comments" do
+  query = Comment.all
+
+  # handle filters
+  error 400, "user_id is required" unless params['user_id']
+  query = query.where(author_id: params['user_id'])
+
+  error 400, "course_id is required" unless params['course_id']
+  query = query.where(course_id: params['course_id'])
+
+  if params["flagged"] == 'True'
+    query = query.where(:abuse_flaggers.nin => [nil, []])
+  end
+
+  # handle pagination
+  total = query.count
+  page = (params['page'] or 1).to_i
+  page_size = (params['page_size'] or 10).to_i
+  query = query.skip((page - 1) * page_size).take(page_size)
+  num_pages = (total.to_f / page_size).ceil
```

Comment:
nit: looking at other places that handle pagination I see a differences in the way pagination is handled. Have a look at the `handle_threads_query` function and see how it does pagination, and apply similar patterns here for consistency.

Reply:
I made it more consistent by using the `DEFAULT_*` values and the `paginate` method instead of `skip`/`take`. Let me know if anything else could be improved here.